### PR TITLE
topic: Added pmmp

### PR DESCRIPTION
**Pull request by thebigsmileXD**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> This topic shows up quite alot when dealing with Minecraft: Bedrock Edition, and with the aliases and related topics it will become alot easier to have an overview of what is happening in the community, and what kind of new plugins have been released. The topic `pmmp` currently has 313 results, so i guess it is already worth adding.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by shook807** [comment]
read me

**Comment by shook807** [comment]
nice

File: topics/pmmp/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (lexuansang93) [inline topics/pmmp/index.md]: 12amgn3ofuKyhRYd2ncFxLZ9NDw7c3mZJN?amount=3

**Comment by shook807** [comment]
Hello.  I don't understand any of this.  I did not intentionally sign up
for GitHub.  I just googled it and found out it a way to spread lies and
fake new and invade others privacy.  So unsubscribe me.  Thank you

**Pull request merged**